crates/topos-tce-api/src/grpc/mod.rs

```diff
@@ -160,15 +160,18 @@ impl ApiService for TceGrpcService {
             }
 
             receiver
-                .map(|value| match value {
+                .map(|value| {
+                    match value {
                     Ok(Ok((position, ref certificate))) => {
```

Comment:
Is indentation off here or is it my diff viewer that is wonky?

Reply:
it is off indeed ! it's due to : https://github.com/rust-lang/rustfmt/issues/3863#issuecomment-700435806